src/renderer/components/+cluster-settings/status.tsx

```diff
@@ -27,6 +36,10 @@ export class Status extends React.Component<Props> {
             </TableRow>
           );
         })}
+        <TableRow key="kubeconfig">
+          <TableCell>Kubeconfig</TableCell>
+          <TableCell className="value" onClick={this.openKubeconfig}><a>{cluster.kubeConfigPath}</a></TableCell>
```

Comment:
FYI: you can use also `@include pseudo-link` mixin in component styles to display as a link.

Reply:
Done